unit-tests/Extension/ExceptionsTest.php

```diff
@@ -143,4 +144,37 @@ public function testExceptionRethrow()
             $this->assertSame(8, $e->getLine());
```

Comment:
Just wonder why line number is 8.

The test says it is 11, I thought it should be 83.

@Jurigag any idea?